src/Mod/Path/PathScripts/PathAdaptiveGui.py

```diff
@@ -45,16 +45,22 @@ def getForm(self):
         formLayout.addRow(QtGui.QLabel("Coolant Mode"), form.coolantController)
 
         # cut region
+
+        form.direction.clear()
         form.Side = QtGui.QComboBox()
-        form.Side.addItem("Inside")
-        form.Side.addItem("Outside")
+        form.side.addItem(
+            QtCore.QT_TRANSLATE_NOOP("Path", "Inside"), "Inside"
```

Comment:
Not for this PR (obviously!) but someday it might be nice to make these Enums instead of strings so that a linter/typechecker can catch mistakes.